_caseStudies/rhl.html

```diff
@@ -1,4 +1,7 @@
 ---
 layout: default
 sitemap: false
+thumbnail: "/assets/images/work/rhlThumbnail.jpg"
+title: RHL.co.uk
+published: "n"
```

Comment:
What does published do? Can we use sitemap: false if it's to stop it showing up?